docs/tutorial_tx_service_set_delegate.md

```diff
@@ -0,0 +1,91 @@
+---
+id: tutorial_tx_service_set_delegate
+title: Set a delegate on the Safe transaction service
+sidebar_label: Set a delegate
+---
+
+The Gnosis Safe transaction service keeps track of transactions sent via Gnosis Safe contracts. Another tutorial shows how to initiate and sign transactions using this service. Initiated transactions will only be show in the official Gnosis Safe clients (Mobile & web) if the transaction was initiated and signed by an owner of the Safe. This is to prevent random accounts spamming transaction lists of Safes.
```

Comment:
Should we link to this "other tutorial"?

Reply:
added